container/health.go

```diff
@@ -19,12 +19,7 @@ func (s *Health) String() string {
 		return types.Unhealthy
 	}
 
-	switch s.Status {
-	case types.Starting:
-		return "health: starting"
```

Comment:
The reason for the original code was that without it you just see the container status as e.g. "Up 20 seconds (starting)" which could be confusing because it's not obvious that it's the health check that's starting, or what the difference is between "Up" and "starting". With separate columns (as in the original patch), "starting" is clear on its own.